ReadMe.md

```diff
@@ -13,33 +13,62 @@ npm install --global alfred-bear
 While creating note templates with static content can be useful, having the possibility to use dynamic placeholder from any source I want is really great.
 That's why I built this workflow.
 
-To get an idea how this should work, take a look at an example: the [Jira Ticket](https://github.com/jmeischner/alfred-bear-jira-ticket)
+To get an idea how this work, take a look at template examples: the [Daily Log](https://github.com/jmeischner/alfred-bear-daily-log) or [Jira Ticket](https://github.com/jmeischner/alfred-bear-jira-ticket).
 
-The idea is that there is a _index.yml_ which contains a list of your templates and some optional variables to configure the template behavior.
+### Configuration
 
-### The Index
+This workflow requires an `index.yml` file which contains a list and configuration of your templates.
 
-the `index.yml` has the following form
+The default workflow configuration directory is `~/.bear-templates` and can be changed by setting the `bearTemplateIndex` variable in the workflow settings.
+The template directory can be synced e.g. with a .dotfiles repository, Dropbox, etc.  
+
+This directory should contain a template index file (default: `index.yml`) and your templates.  
+
+An example directory structure:
+
+```
+├──~/.bear-templates
+│    ├── index.yml
+│    ├── weekly-review
+│    │   ├── template.md
+│    │   └──script.js
+│    ├── jira-ticket
+│    │   ├── ticket.md
+│    │   ├── ticket.js
+│    │   ├── package.json
+│    │   ├── node_modules
+│    │   ├── diary.md
+```
+
+To create the `index.yml` in the expected default directory:
+
+```shell
+mkdir ~/.bear-templates/
+touch ~/.bear-templates/index.yml
+```
+
+### The index.yml
+
+An example of `index.yml` content:
 
 ```yml
 templates:
-  - title: "Weekly Review"
-    file: "weekly-review/template.md"
-    script: "weekly-review/script.js"
-    newWindow: true
+  - title: "Daily Log"
+    file: "alfred-bear-daily-log/Daily Log.md"
+    script: "alfred-bear-daily-log/script.js"
+    newWindow: false
```

Comment:
I've changed this as the `Weekly Review` workflow is not listed but the `Daily Log` is and it is very easy to setup.
The Jira Plugin is more complex to setup so didn't used that one.

I think it makes more sense to use this as an example considering it's mentioned elsewhere in the readme.